content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -43,11 +43,12 @@ this is true when user namespaces are used.
 * You need to be able to exec into pods
 * Feature gate `UserNamespacesStatelessPodsSupport` need to be enabled.
 
-In addition, support is needed in the
+In addition, support is **needed** in the
```

Comment:
I don't think this PR is the right place to add that emphasis, and I also don't think it's right to emphasize anyway. The word “needed” is already clear.

If we did make a change, I'd prefer it to align with the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) - eg, [use the active voice](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice) - and arrive in a separate PR.